lightning/backend/tidb.go

```diff
@@ -263,7 +265,15 @@ func (be *tidbBackend) WriteRows(ctx context.Context, _ uuid.UUID, tableName str
 	}
 
 	var insertStmt strings.Builder
-	insertStmt.WriteString("INSERT INTO ")
+	switch be.OnDuplicate {
```

Comment:
what's the default behavior of  `OnDuplicate`?

Reply:
should we give it a default value if users don't configure `OnDuplicate`

Reply:
default behavior is replacing on duplicate, which specifies in `Adjust()`.

https://github.com/pingcap/tidb-lightning/pull/243/files#diff-5f8541d99c1c66dcc6e24f4418ce5c24R205

`Adjust()` will be called in `RunOnce` and `handlePostTask()` (which then goes to RunServer), cover all user's case.

Codes call `NewTiDBBackend` should specify this param manually 🤔

Reply:
IMO the extra defense is redundant. Not a big issue anyway.